src/pages/nutrition/table.jsx

```diff
@@ -0,0 +1,160 @@
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TextField from "@mui/material/TextField";
+import SelectAutoWidth from "./selectComp";
+import { Box } from "@mui/material";
+import Checkbox from "@mui/material/Checkbox";
+import AdditionalNutriments from "./additionalNutritions";
+
+function createData(
+  label, property, unit
+) {
+  return { label, property, unit };
+}
+
+export default function NutritionTable() {
+
+  const [nutriments, setNutriments] = React.useState([
+    {
+      off_nutriment_id: "energy_kj",
+      label: "Energie (kJ)",
+      value: "",
+      unit: "null",
+      quantification: "=",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Protein",
+      value: "",
+      unit: "null",
+      quantification: "<",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Shugar",
+      value: "",
+      unit: "null",
+      quantification: "<",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Fat",
+      value: "",
+      unit: null,
+      quantification: "<",
+      robotoffPrediction: null
+    }
+  ])
+
+  const [additionalNutriments, setAdditionalNutriments] = React.useState([
+    {
+      off_nutriment_id: "energy_kj",
+      label: "Energie (kJ)",
+      value: "",
+      unit: "null",
+      quantification: "=",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Protein",
+      value: "",
+      unit: "null",
+      quantification: "<",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Shugar",
+      value: "",
+      unit: "null",
+      quantification: "<",
+      robotoffPrediction: null
+    },
+    {
+      off_nutriment_id: "energy_kcal",
+      label: "Fat",
+      value: "",
+      unit: null,
+      quantification: "<",
+      robotoffPrediction: null
+    }
+  ])
+
+  function onchangeHandler(e) {
+    const {value, name} = e.target
+    setNutriments(prevState => prevState.map(
+      nutr => {
+        return name === nutr.label? {...nutr, value} : nutr
+      }
+    ))
+  }
+
+  const rows = nutriments.map(nutrition => {
+    return (
+      createData(<Box sx={{
+          display: "flex",
+          alignItems: "center",
+          flexDirection: "row"
+        }}><TextField id="outlined-basic"
+                      type={'number'}
+                      label={nutrition.label}
+                      variant="outlined"
+                      sx={{ width: "10rem" }}
+                      value={nutrition.value}
+                      name={nutrition.label}
+                      onChange={onchangeHandler}
+        /> , <SelectAutoWidth /></Box>,
+        <Checkbox sx={{}} />));
```

Comment:
is `sx={{}}` needed?

Reply:
I removed checkbox and leaved delete icon. Still working on it